test/tools/ossfuzz/SolidityCustomMutatorInterface.h

```diff
@@ -0,0 +1,54 @@
+/*
+	This file is part of solidity.
+
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+// SPDX-License-Identifier: GPL-3.0
+/**
+ * Implements libFuzzer's custom mutator interface by making a call
+ * to the Solidity custom mutator, defaulting to libfuzzer's
+ * default mutator when the former fails.
+ */
+
+#pragma once
+
+#include <liblangutil/Exceptions.h>
+
+#include <cstddef>
+#include <cstdint>
+#include <ostream>
+#include <random>
```

Comment:
I think `<random>` is unused in this module.

Reply:
I also think `random` can be removed

Reply:
Actually all of them